.gitattributes

```diff
@@ -0,0 +1,2 @@
+*.conf            text   eol=lf
+*.sh              text   eol=lf   diff=bash
```

Comment:
I don't understand the `diff=bash` part. Could you clarify for my understanding, please?

Reply:
Sorry, I copied it from somewhere else. It was meant to make git diffs more useful (like ignoring the number of spaces). But it turns out we would also need to define a custom diff driver `[diff "bash"]`, which looks like black magic and actually does not help for such small `*.sh` files.

I'll remove the `diff` part.